lib/centralisation/class/module.dart

```diff
@@ -0,0 +1,67 @@
+import "dart:convert";
+
+class Module {
+  Module({
+    required this.name,
+    required this.description,
+    required this.icon,
+    required this.url,
+    this.liked,
+  });
+  late final String name;
+  late final String description;
+  late final String icon;
+  late final String url;
+  late final bool? liked;
+
+  Module.fromJson(Map<String, dynamic> json) {
+    name = utf8.decode(latin1.encode(json['name']), allowMalformed: true);
+    description = utf8.decode(latin1.encode(json['description']), allowMalformed: true);
+    icon = json['icon'];
+    url = json['url'];
+    liked = json['liked'];
+  }
+
+  Map<String, dynamic> toJson() {
```

Comment:
You are not sending data to Hyperion or any other backend, thus you don't need to have a toJson function, because its only goal is to format data to send it